src/plugins/intel_gpu/src/kernel_selector/kernels/space_to_batch/space_to_batch_kernel_base.cpp

```diff
@@ -70,9 +74,26 @@ JitConstants SpaceToBatchKernelBase::GetJitConstants(const space_to_batch_params
         }
     };
 
-    makeJitConstForParam(jit, "BLOCK_SHAPE", params.block_shape, 1);
-    makeJitConstForParam(jit, "PADS_BEGIN", params.pads_begin, 0);
-    makeJitConstForParam(jit, "PADS_END", params.pads_end, 0);
+    if (params.block_type == base_params::ArgType::Input || params.has_dynamic_tensors()) {
```

Comment:
hmm these kernels do not have dynamic impl yet but why can param have dynamic tensor?

Reply:
deleted